docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -1272,10 +1286,12 @@ def map_raw_to_final_display(label, document_type="pdf", reserved_whole_words=ge
 
   if (label in reserved_whole_words
    or label in reserved_var_plurals
-   or label in undefined_person_prefixes):
+   or label in undefined_person_prefixes
+   or label in custom_people_plurals_map.values()):
      return label
 
   # Break up label into its parts: prefix, digit, the rest
+  all_prefixes = reserved_prefixes + list(custom_people_plurals_map.values())
```

Comment:
`all_prefixes` doesn't look like it used?

Reply:
Good eye. I think it was supposed to go into line 1295: `label_groups = get_reserved_label_parts(<strong>all_prefixes</strong>, label)`. Obviously the tests I ran were not sufficient to catch all the edge cases.

I'll fix that up 👍

Reply:
Edited on the line below.